src/garage/torch/algos/ppo.py

```diff
@@ -83,17 +80,13 @@ def _compute_objective(self, advantages, valids, obs, actions, rewards):
             torch.Tensor: Calculated objective values
 
         """
-        # pylint: disable=unused-argument
         # Compute constraint
         with torch.no_grad():
             old_ll = self._old_policy.log_likelihood(obs, actions)
         new_ll = self.policy.log_likelihood(obs, actions)
 
         likelihood_ratio = (new_ll - old_ll).exp()
 
-        # Memorize the policy state_dict
-        self._old_policy.load_state_dict(self.policy.state_dict())
```

Comment:
Why have you made changes to PPO, was there some issue?

Reply:
`TRPO` and `PPO` both needs to use self._old_policy, so I moved the common code `vpg.py`